src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -407,17 +378,21 @@ class F extends GPUTest {
         var attributesInBounds : bool = ${attributeNames
           .map(a => `validationFunc(attributes.${a})`)
           .join(' && ')};
-        var indexInBounds : bool = VertexIndex == 0u ||
+
+        var indexInBoundsCountFromBaseVertex : bool =
             (VertexIndex >= ${vertexIndexOffset}u &&
             VertexIndex < ${vertexIndexOffset + numVertices}u);
+        // For D3D12, VertexIndex count from 0. For index robust access it can be 0.
```

Comment:
I don't think we should have this D3D12-specific logic. It will be fixed eventually. Instead we'll just add a suppression in Chromium for the failing tests.